src/napari_skan/napari_skan.py

```diff
@@ -0,0 +1,13 @@
+import numpy as np
+from enum import Enum
+from skimage.morphology import skeletonize
+
+class SkeletonizeMethod(Enum):
+    Zhang = "zhang"
+    Lee = "lee"
+
+
+def skeletonize_labels(labels: "napari.types.LabelsData", method: SkeletonizeMethod) -> "napari.types.LabelsData":
+    binary_labels = (labels > 0).astype(np.uint8)
+    skeletonized = skeletonize(binary_labels, method=method)
```

Comment:
@jamesyan-git yeah this was super tricky and confusing! But I finally cracked it! 😅 

```suggestion
    skeletonized = skeletonize(binary_labels, method=method.value)
```

What's happening is that skimage doesn't understand the enum method, so it finds no valid method and ends up in the else clause. It's a confusing error message, because apparently it never occurred to us that one could pass in an invalid argument. 😂 Would you mind raising an issue in skimage? I get the same error with:

```python
import numpy as np
from skimage.morphology import skeletonize

result = skeletonize(np.ones((5, 5)), method='foo')
```